packages/drivers/debugger/src/fluidDebuggerUi.ts

```diff
@@ -232,7 +233,8 @@ export class DebuggerUI {
 			() => {
 				const files = fileSnapshot.files;
 				if (files) {
-					controller.onSnapshotFileSelection(files[0]);
+					// Why are we non null asserting here
```

Comment:
?

Reply:
These are for the area owners on advising why we should be non null asserting or what we should do in the follow up PR to do the code change correctly. If there is no appropriate comment we will settle on a standardized comment for places where the expectation is the array is dense and you can see the code performs bounds checks (rather than just 'TODO: why is this non-null assertion ok' or some equivalent)